app/src/main/java/co/develhope/meteoapp/adapter/SpecificDayAdapter.kt

```diff
@@ -0,0 +1,123 @@
+package co.develhope.meteoapp.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import co.develhope.meteoapp.R
+import co.develhope.meteoapp.databinding.SpecificdayCardBinding
+import co.develhope.meteoapp.databinding.SpecificdayHourlyBinding
+import co.develhope.meteoapp.databinding.SpecificdayTitleBinding
+import co.develhope.meteoapp.model.SpecificDayModel
+
+
+
+
+class SpecificDayAdapter ( private val context: Int, private val list: List<SpecificDayModel>):
```

Comment:
un context che è un Int, cosa è?

Reply:
perché vado a richiamare un intero non una stringa in SpecificDay(frammento) e senza mi da errore infatti